config/product/explorer.js

```diff
@@ -209,6 +215,17 @@ export function init(store) {
     overview:    true,
   });
 
+  virtualType({
+    label:       'Cluster Members',
```

Comment:
Localisation

Reply:
Switched this label and the label of the other virtual types.